support-modules/aws/src/main/scala/com/gu/aws/AWSAsyncHandler.scala

```diff
@@ -1,25 +1,24 @@
 package com.gu.aws
 
 import java.util.concurrent.{Future => JFuture}
-
 import com.amazonaws.handlers.AsyncHandler
 import com.amazonaws.{AmazonServiceException, AmazonWebServiceRequest}
-import com.gu.monitoring.SafeLogger
+import com.typesafe.scalalogging.LazyLogging
 
 import scala.concurrent.{Future, Promise}
 
 class AwsAsyncHandler[Request <: AmazonWebServiceRequest, Response](f: (Request, AsyncHandler[Request, Response]) => JFuture[Response], request: Request)
-    extends AsyncHandler[Request, Response] {
+    extends AsyncHandler[Request, Response] with LazyLogging{
```

Comment:
do we even use sentry for server side Error reporting nowadays?  I have never looked at it!

Reply:
Me neither so I would guess not. We could probably remove it